src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+        if (user != null && user.getPassword() == password) {
+            return true;
+        }
         return false;
```

Comment:
You can simplify your code:
```suggestion
        return user != null && user.getPassword() == password;
```